core/components/handler/pktStorage.go

```diff
@@ -145,6 +147,7 @@ func (e pktEntry) MarshalBinary() ([]byte, error) {
 	rw := readwriter.New(nil)
 	rw.Write(e.AppEUI)
 	rw.Write(e.DevEUI)
+	rw.Write(uint16(e.FPort))
```

Comment:
This line makes it difficult to deploy this feature to our _staging_ environment, as existing databases can still contain data in the old format. This means we have three options:

- Don't deploy this to staging
- Delete all old data and deploy to staging
- Write something that migrates old data

I vote for option 2, as it is the simplest solution. There aren't that many scheduled packets that would get lost